students/brandon_nguyen/session03/mailroom.py

```diff
@@ -0,0 +1,114 @@
+import sys
+import unittest
+from decimal import Decimal
+
+#!/usr/bin/env python3
```

Comment:
This line should be always the first line of any `.py` file (if you want to put it), even before all `import` statements.